### ISSUE TITLE
Details List: Button not reacting to Enter KeyPress

### ISSUE THREAD

**Issue by bhaveshbhati** [OfficeDev/office-ui-fabric-react#1529]
### Bug Report

- __Package version(s)__: 2.10.4
- __Browser and OS versions__: Chrome

#### Priorities and help requested (not applicable if asking question):

Are you willing to submit a PR to fix? (Yes, No)

Requested priority: (Blocking, High, Normal, Low) 

Products/sites affected: (if applicable)

#### Describe the issue:

In Details List , I am using OfficeFabric button but it is not reacting to Enter keypress on it.
I have a onClick handler which gets called when I click on the button but not when I press Enter on it.

#### Actual behavior:

Nothing happens when I press Enter.

#### Expected behavior:

It should have called the onClick handler when I press Enter.

### If applicable, please provide a codepen repro:

http://codepen.io/anon/pen/PmqBpq
